#7 - Allow the user to step through the battle turn-by-turn

tenshi13 opened the pull request:
- rejig test-battle.js logic to allow press enter to continue turn
- added helper function clearScreen (clears cli)

this is my attempt to resolve #7 
@Asuza - sorry... I have moved the code a fair bit... 
do tell if I've changed it too much, I was having trouble with the stdin

tenshi13 committed:
- added helper function clearScreen (clears cli)
- rejig test-battle.js logic to allow press enter to continue turn

Asuza commented on helpers.js:
```diff
@@ -246,4 +246,9 @@ helpers.findNearestTeamMember = function (gameData) {
 };
 
+// Clears the console's screen
+helpers.clearScreen = function() {
```
I can see why you put `clearScreen` in this file, but would you please move it to _test-battle.js_? The _hero.js_ and _helper.js_ files should be completely modifiable by the player, so _test-battle.js_ shouldn't depend on them for much more than what move the hero is going to make.

tenshi13 replied:
Yeah man, sorry wasn't very sure where to put it, I'll move it back there then

Asuza commented on test-battle.js:
```diff
@@ -35,56 +49,127 @@ var enemyMoveFunction = function (gameData, helpers) {
     }
     console.log('-----');
-    console.log('Turn ' + i + ':');
+    console.log('Turn ' + turn + ':');
```
Nice change!

tenshi13 replied:
wooo very happy you liked it : D

Asuza commented:
It's very good overall! Please run it through

```
./node_modules/eslint/bin/eslint.js ./test-battle.js 
```

when you're done making that change. Thanks!

tenshi13 committed:
Issue#7 - minor changes
- moved clearScreen to test-battle.js
- did eslint.js and updated recomended syntax changes
